app/imports/components/Users/AdminPanel/UsersManagement/ProjectList/projectList.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import Subheader from 'material-ui/Subheader';
+import { List, ListItem, makeSelectable } from 'material-ui/List';
+import { PropTypes } from 'prop-types';
+
+
+export class ProjectList extends React.Component {
+
+    constructor(props) {
+        super(props);
+        this.SelectableList = makeSelectable(List); // @TODO Do it like in tutorial
```

Comment:
You should do it like in tutorial, because we will get reusable component to do this.